src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -81,7 +85,18 @@ case class OTreeIndex(index: TahoeLogFileIndex) extends FileIndex {
     }.toArray
     val stagingStats = stagingFiles
     val fileStats = qbeastFileStats ++ stagingStats
-
+    val execId = spark.sparkContext.getLocalProperty(SQLExecution.EXECUTION_ID_KEY)
+    partitionFilters.foreach(f => {
```

Comment:
I think it's best if we print the sequence directly in a single message.

Reply:
Ok, modified to print the sequence as a single message.